api/labels/labels_api_mocked.go

```diff
@@ -383,7 +383,7 @@ func RemoveLabelMocked(t *testing.T, labelIn *types.Label) {
 
 	// call service
 	resourceID := "5b5074735f7c880ad9c6bbce"
-	cs.On("Delete", fmt.Sprintf("v1/labels/%s/resources/%s/%s", labelIn.ID, labelIn.ResourceType, resourceID)).Return(dIn, 204, nil)
+	cs.On("Delete", fmt.Sprintf("/v2/labels/%s/resources/%s/%s", labelIn.ID, labelIn.ResourceType, resourceID)).Return(dIn, 204, nil)
```

Comment:
Why have you changed v1 for /v2?

Reply:
Well, it should be "v2" and the slash is included due to normalization, canonical way should be.
The approach now,  trying to normalize the url/paths. every path should have and initial slash.

And for base domain, it could be provided by definition or not; but in any case, i applied piece of code to remove if exist at the end of base URL.
So with this way, the URLs are well formed and not repeating // in the path